infra/examples-dev/aws-google-workspace/main.tf

```diff
@@ -249,9 +249,10 @@ module "aws-psoxy-long-auth-connectors" {
   function_parameters            = each.value.secured_variables
   todo_step                      = module.source_token_external_todo[each.key].next_todo_step
 
-  environment_variables = {
+  environment_variables =  merge(each.value.environment_variables,
```

Comment:
sorry! bad merge or copy-paste.   I need to port this over to `v0.4.8-rc`, which has refactored a bunch of this stuff; but in meantime we should indeed merge to `main` so anyone gets it ASAP in their examples